.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     # 'machine' executor runs Unit tests ~x1.5 faster, comparing to 'docker' executor
     # but the fastest is still ~x1.5-2 slower, comparing to Travis
     machine: true
-    parallelism: 4
+    parallelism: 6
```

Comment:
That's a different job `unit` which is currently unused in CircleCI.
Find the `packages` job in CircleCI yaml metadata.